server/README.md

```diff
@@ -0,0 +1,181 @@
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+}
+```
+
+#### GetReferenceData
+
+Get the specific TEE's OPA `Reference Data(.json)`. The message:
+```PROTO
+message GetReferenceDataRequest {
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+}
+```
+
+#### SetPolicy
+
+Set the specific TEE's OPA `Policy(.rego)`. It can make the Attestation Server evaluate the TEE TCB status according to specific user's preference. And it will echo error if the new `Policy(.rego)` syntax is illegal. The message:
+```PROTO
+message SetPolicyRequest {
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+    // The "Policy(.rego)" file's content.
+    bytes content = 3;
+}
+```
+
+#### SetReferenceData
+
+Set the specific TEE's OPA `Reference Data(.json)`. It can attach a new released program's reference measurement value into the corresponding program's allow list. The message:
+```PROTO
+message SetReferenceDataRequest {
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+    // The "Reference Data(.json)" file's content.
+    bytes content = 3;
+}
+```
+
+#### RestoreDefaultPolicy
+
+Restore the specific TEE's OPA `Policy(.rego)` to default value. The message:
+```PROTO
+message ResetPolicyRequest {
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+}
+```
+
+#### RestoreDefaultReferenceData
+
+Restore the specific TEE's OPA `Reference Data(.json)` to default value. The message:
+```PROTO
+message ResetReferenceDataRequest {
+    // Supported TEE types
+    Tee tee = 1;
+    // Optional: Designate the user id. It should be kept as "None" currently.
+    optional string user = 2;
+}
+```
+
+#### TestOpa
+
+It's main purpose is to ensure the updated `Policy(.rego)` or `Reference Data(.json)` can work as expected before upload to remote Attestation Server. The message:
+```PROTO
+message TestOpaRequest {
+    // The OPA "Policy(.rego)" content that need to test.
+    bytes policy = 1;
+    // The "Reference Data(.json)" content that need to test.
+    bytes reference = 2;
+    // The "input" which will be evaluated by the request's "policy" and "referece".
+    bytes input = 3;
+}
+```
+
+## Usage
+
+Here are the steps of building and running Attestation Server:
+
+### Build
+
+Build Attestation Server:
+```shell
+git clone https://github.com/confidential-containers/attestation-service
+cd attestation-service
+cargo build --release
+```
+
+### Run
+
+- For help information, run:
+```shell
+./attestation-server --help
```

Comment:
Running `cargo build --release` creates binary inside `target/release`. So I guess it should be

`target/release/attestation-server --help`